libdnf/base/goal.cpp

```diff
@@ -621,8 +618,6 @@ GoalProblem Goal::Impl::add_reinstall_to_goal(const std::string & spec, GoalJobS
         }
     }
 
-    // TODO(jmracek) Implement fitering from_repo_ids
```

Comment:
Please could you revert the change? It is note to implement reinstall packages installed from a repository.